src/System.Globalization/ref/System.Globalization.cs

```diff
@@ -8,6 +8,7 @@
 
 namespace System.Globalization
 {
+    // ToDo: Calendar should be removed from here once it is available in System.Runtime
```

Comment:
Are you tracking this todo with an issue? Another way to handle this is to temporarily do a P2P reference to the new System.Runtime.